src/telemetry/telemetry.c

```diff
@@ -937,6 +1025,17 @@ build_telemetry_report()
 	add_replication_telemetry(parse_state);
 	pushJsonbValue(&parse_state, WJB_END_OBJECT, NULL);
 
+	key.type = jbvString;
+	key.val.string.val = REQ_ACCESS_METHODS;
+	key.val.string.len = strlen(REQ_ACCESS_METHODS);
+	(void) pushJsonbValue(&parse_state, WJB_KEY, &key);
+	add_query_result_dict(parse_state,
+						  "select amname as name, sum(relpages) as pages, count(*) as "
+						  "instances "
+						  "from pg_class join pg_am on relam = pg_am.oid where amname = "
```

Comment:
Why does it filter on `amname = 'btree'`?

Reply:
Leftover from debugging. Removed.

Reply:
Also added test that both btree and heap exists, since these are always present.